.github/issue_labels/labels.csv

```diff
@@ -0,0 +1,112 @@
+Label name,Issues,Description,SME,Team,Platform,Blanks,Issue count
```

Comment:
🤔 I assume this is taken from the Google doc, verbatim.
I think we should have a single source of truth, and it should be located in the repo, making the doc obsolete.